src/_common/analytics/analytics.service.ts

```diff
@@ -52,6 +53,13 @@ export class Analytics {
 		});
 
 		EventBus.on('routeChangeAfter', () => {
+			// If the home route is being visited and the user is not signed in,
+			// we want to track the event as '/discover' instead of the user feed route.
+			if (router.currentRoute.fullPath === '/' && !this.appUser) {
```

Comment:
any specific reason to use fullPath and not just path?
fullPath seems to include the query parameters and hash.

Reply:
`path` does seem more correct here then - changed